boilerplate/app/navigation/root-navigator.tsx.ejs

```diff
@@ -1,11 +1,19 @@
 import React from "react"
 import { NavigationContainer, NavigationContainerRef } from "@react-navigation/native"
-import { createNativeStackNavigator } from "@react-navigation/native-stack"
-
+<% if(props.useExpo){ %>
+  import { createStackNavigator } from "@react-navigation/stack"
+<% } else{ %>  
+  import { createNativeStackNavigator } from "react-native-screens/native-stack"
+<% } %>
 import { RootParamList } from "./types"
 import { PrimaryNavigator } from "./primary-navigator"
 
-const Stack = createNativeStackNavigator<RootParamList>()
+<% if(props.useExpo){ %>
+  const Stack = createStackNavigator<RootParamList>()
```

Comment:
I've added your changes from this PR to a project I created last night. Everything seems to work, but I am getting a type error from line 16/24 below because `stackPresentation` doesn't exist on `Stack.Navigator` for `@react-navigation/stack`.